src/JHipsterNetSampleApplication/Controllers/OperationController.cs

```diff
@@ -70,8 +75,11 @@ public ActionResult<IEnumerable<Operation>> GetAllOperations(IPageable pageable)
         public async Task<IActionResult> GetOperation([FromRoute] long id)
         {
             _log.LogDebug($"REST request to get Operation : {id}");
-            var result =
-                await _applicationDatabaseContext.Operations.SingleOrDefaultAsync(operation => operation.Id == id);
+            var result = await _applicationDatabaseContext.Operations
+                .Include(operation => operation.BankAccount)
+                    .ThenInclude(bankAccount => bankAccount.User)
+                .Include("OperationLabels.Label")
```

Comment:
Either we use the lambda notation (ie. `operation => operation.BankAccount`) or the string one (ie. `"OperationLabels.Label"`) but please choose one. We must stay consistent.

Reply:
We are forced to do so on this property "OperationLabels". That's the only way to fetch the Labels from the database in a query.
It's a bit tricky but the field is private to hide the join entity to the user and make its management more transparent. So to access it we can only call it in a string argument. Moreover, we can't call the Labels field in the query because it is not mapped to the database.
If you prefer we can turn it to public for consistency using lambda notation and add a [JsonIgnore] annotation to hide it to the frontend ?

Reply:
Several options (sorted by preference)
- you turn it public 
- define constants `LABEL_ENTITY_NAME = "OperationLabels.Label"`& `OPERATION_ENTITY_NAME = "OperationLabels.Operation"` in the join entity
- compute it by reflection
- you define generic constants in the Join Entity (I don't like this approach)

Reply:
Ok I turned it to public ;)